hangman.rb

```diff
@@ -0,0 +1,96 @@
+def select_word
+    words = ["CHOCOLATE", "GERMANY", "RADIO", "WEBDESIGN"]
+
+    sample_word = words.sample
+
+    #puts sample_word
+end
+
+def word_hint
+    if @game_word == "CHOCOLATE"
+        hint = "A sweet"
```

Comment:
Please follow this for Indentations: https://rubystyle.guide/#spaces-indentation